### DISCUSSION TITLE
pack APK

### DISCUSSION THREAD

**Discussion by firemen01** [DelphiWorlds/Kastri#251] [Q&A]
tell me where the compression level of the APK is configured?
On 12.1 my application is under 20 megabytes, on 12.2 over 50. I checked the insides - all the files are almost the same size.
I unpacked the APK and packed it with a zip archiver with compression level 1, the archive size is under 23 m. and here it is over 50. I can't imagine who packs it like that.

**Comment by firemen01** [comment]
empty application if developer then about 11 meters, if appstore I sign then 26 (without 64, only 32)! meters
how to deal with this?

**Comment by DelphiWorlds** [comment]
Please read [this link re "Seeking Solutions"](https://github.com/DelphiWorlds/HowTo/blob/main/Readme.md#seeking-solutions). The discussion area here is related to Kastri, not for general questions.
